### DISCUSSION TITLE
p-menu  and galleria indicator not showing correctly after updating to version 17.3.2

### DISCUSSION THREAD

**Discussion by faentu** [primefaces/community#948] [PrimeNG]
I have updated primeng from version 16.3.1 to 17.3.2 as I have migrated angular to version 17 and now p-menu looks a little bit different and the indicator for the galleria component has changed from a circle to a square. Do I miss something I have to change?
![primeng_indicators](https://github.com/primefaces/community/assets/60216363/3e04909a-2863-4582-aa99-91e2713e34e8)
![primeng_p-menu](https://github.com/primefaces/community/assets/60216363/804decb6-f62d-4ebd-a015-afcfc29e0677)

**Comment by cetincakiroglu** [comment]
Could you please share a stackblitz example so we can identify the issue clearly?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity
